helpers/deproxy.py

```diff
@@ -450,6 +456,29 @@ def create(method, headers, uri='/', version='HTTP/1.1', date=False,
         msg = HttpMessage.create(first_line, headers, date=date, body=body)
         return Request(msg)
 
+class H2Request(Request):
```

Comment:
No need to do anything in this PR, but the header comment has TODO about why `Reqeust` is implemented on our own instead of using a standard `HTTPRequest`? If there is really nothing specific about our own implementation and we can migrate to a standard implementation, then let's create an infrastructure issue for this - this could be a good 'good to start' issue. After that the TODO comment can be removed. If we do need our own implementation, then it's worth commenting what is specific about our implementations.

Reply:
In python3 we must use `http.client`. But I believe we can't migrate to `http.client` without large amount of changes that affect DeproxyClient and DeproxyServer, as well some tests in which we compare Request and Response. `http.client` doesn't have separated `Request` class, it combines `Request` and `Connection` in `HTTPConnection` class, but in our framework we use Request class in DeproxyServer and DeproxyClient like a wrapper for request message.